src/chrome/content/rules/LastPass.xml

```diff
@@ -1,11 +1,8 @@
-<ruleset name="LastPass.com" default_off="Breaks addon ( https://www.eff.org/r.3Dq )">
+<ruleset name="LastPass.com">
   <target host="lastpass.com" />
   <target host="*.lastpass.com" />
 
   <securecookie host="^(?:.+\.)?lastpass\.com$" name=".*"/>
 
-  <rule from="^http://(?:www\.)?lastpass\.com/" to="https://lastpass.com/"/>
-
-	<rule from="^http://forums\.lastpass\.com/"
-		to="https://forums.lastpass.com/" />
+  <rule from="^http:" to="https:" />
```

Comment:
I've actually used Wolfram Alpha to find the sub-domains. I've tested all the sub-domains which were listed and they all worked. One sub-domain, download-do.lastpass.com, gave a 404, but it has nothing to do with http or https.